xrpl/models/requests/ledger_entry.py

```diff
@@ -264,6 +264,8 @@ class LedgerEntry(Request, LookupByLedgerRequest):
     binary: bool = False
     nft_page: Optional[str] = None
     """Must be the object ID of the NFToken page, as hexadecimal"""
+    include_deleted: Optional[bool] = None
```

Comment:
Why does the xrpl-py client library not contain the `deleted_ledger_index` parameter? I observed it in the javascript client library

Reply:
Since there's no response class in ledger_entry.py so I assume we don't need it?  For js one there's already a response class(interface) so I just add `deleted_ledger_index` to that class.

Reply:
Yes, Python doesn't have response models.